net/pfSense-pkg-Avahi/pkg-plist

```diff
@@ -1,7 +1,6 @@
 pkg/avahi.xml
-pkg/avahi.inc
+pkg/avahi/avahi.inc
```

Comment:
Why move .inc file to a separate directory?

Reply:
Same pattern used in other packages, which allows the introduction of additional files in the future. Same as with the nut and lldpd packages.